README-DEPLOY.md

```diff
@@ -128,6 +128,30 @@ You can find the open-source project and the instructions to deploy it here: [ma
 
 ![Power Tuner UI](https://github.com/mattymoomoo/aws-power-tuner-ui/blob/master/imgs/website.png?raw=true)
 
+## Option 6: deploy the SAR app with Terraform
+
+Simply add the `aws_serverlessapplicationrepository_cloudformation_stack` resource below to your Terraform code and deploy as usual through `terraform apply`.
+
+```hcl
+resource "aws_serverlessapplicationrepository_cloudformation_stack" "lambda-power-tuning" {
+  name           = "lambda-power-tuner"
+  application_id = "arn:aws:serverlessrepo:us-east-1:451282441545:applications/aws-lambda-power-tuning"
```

Comment:
Does this deploy the latest version by default? Can you define the `SemanticVersion` explicitly as well?

Reply:
Yes it does. I updated to add a commented-out version as an example.